numpy/linalg/linalg.py

```diff
@@ -1810,7 +1810,7 @@ def det(a):
 
 # Linear Least Squares
 
-def lstsq(a, b, rcond=-1):
+def lstsq(a, b, rcond="warn"):
```

Comment:
True, I don't feel much either way. Thought the main point of NoValue was for thing like `keepdims` where there is no other option, but its a fair point that it hinders someone from thinking that it is permanent API. Do we use this consistently now also for this use case?